#
# This patch removes an inapropriate assert.
#
# This assert is inappropriate, because the length of the exported name
# object may be smaller than input_name_buffer->value. This has caused
# aborts in RPCSEC_GSS.
#
# This patch is to accommodate Solaris RPC implementation and is not suitable
# upstream contribution.
# Patch source: in-house
#
--- a/src/lib/gssapi/krb5/import_name.c
+++ b/src/lib/gssapi/krb5/import_name.c
@@ -300,7 +300,6 @@ krb5_gss_import_name(minor_status, input_name_buffer,
                     goto fail_name;
                 cp += length;
             }
-            assert(cp == end);
         } else {
             status = GSS_S_BAD_NAMETYPE;
             goto cleanup;
